### ISSUE TITLE
May I know your hyper-parameters of "ssd_lite_mobilenetv1"

### ISSUE THREAD

**Issue by haoyango** [ShuangXieIrene/ssds.pytorch#58]
First,thanks for your great work!
Now,I have trained "ssd_lite_mobilenetv1" on voc data for several times,but the results are not so good,I found the loss curve fluctuate a little violently,so May I know your learning-rate
,weight_decay and warm_up_epochs?

**Comment by foreverYoungGitHub** [comment]
Hi, the detial of the hyperparameter is in this file: https://github.com/ShuangXieIrene/ssds.pytorch/blob/master/experiments/cfgs/ssd_lite_mobilenetv1_train_voc.yml

Let me know if you have any other questions.
